### PR TITLE
checkbox component

### LINKED ISSUE #203: Checkbox component
https://app.zeplin.io/project/595a9cd3b401bf1876faab27/screen/59ee44fef0a38ca9a6149182
https://app.zeplin.io/project/595a9cd3b401bf1876faab27/screen/59ee44fc0c2c05bfc02333f3

image

image

### PR DESCRIPTION
**Pull request by fczuardi**
affects: @crave/farmblocks-input-checkbox

- [x] closes  #203

### CODE REVIEW

**Comment by fczuardi** [comment]
@vNakamura yes, and its vertical distance is bigger than the reference too...

**Comment by fgameiro** [comment]
In Firefox 57 (Ubuntu) the tooltip is bit more misaligned:

disabled-checkbox

**Comment by fczuardi** [comment]
@vNakamura good catch

**Comment by fczuardi** [comment]
readability refactor submitted

File: packages/input-checkbox/src/styledComponents/Checkbox.js [hunk 1]
```diff
@@ -1,32 +1,39 @@
...
+      background-color: ${colors.GREY_16};
+      border-color: ${colors.GREY_16};
+    `;
+  }
+  return css`
+    color: ${props.checked ? "#FFFFFF" : colors.INDIGO_MILK_CAP};
+    background-color: ${props.checked ? colors.INDIGO_MILK_CAP : "#FFFFFF"};
+    border-color: ${props.checked ? colors.INDIGO_MILK_CAP : colors.GREY_16};
+  `;
+};
```
Comment (alcferreira) [inline packages/input-checkbox/src/styledComponents/Checkbox.js]: 😍

**Pull request merged**